spec.html

```diff
@@ -507,15 +511,7 @@ <h1>Runtime Semantics: ClassFieldDefinitionEvaluation</h1>
       1. Perform MakeMethod(_initializer_, _homeObject_).
     1. Else,
       1. Let _initializer_ be ~empty~.
-    1. If _key_ is a Private Name,
-      1. Let _enumerable_ be *false*.
-      1. Let _configurable_ be *false*.
-      1. Let _writable_ be *false*.
-    1. Else,
-      1. Let _enumerable_ be *true*.
-      1. Let _configurable_ be *true*.
-      1. Let _writable_ be *true*.
-    1. Let _desc_ be the PropertyDescriptor{[[Value]]: _initializer_, [[Writable]]: _writable_, [[Enumerable]]: _enumerable_, [[Configurable]]: _configurable_}.
+    1. Let _desc_ be ! DefaultMethodPropertyDescriptor(_key_, _initializer_, *true*).
```

Comment:
The name DefaultMethodPropertyDescriptor feels weird here because it's used for fields, but the name of the operation includes method.

Reply:
Gotcha; I'll rework it a bit.

Reply:
k, i ended up abandoning the idea of this last commit since there's not tons of value from the abstract op.